llvm_ir_dataset_utils/compile_time_analysis_tools/python_processing/fitting_and_plotting.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: ascii -*-
+import numpy as np
+import matplotlib.pyplot as plt
+from read_column import open_and_load
+# Usage:
+# from fitting_and_plotting import plot_functionality
+
+
+def plot_functionality(lang: str, show: bool = False) -> None:
+    '''
+    Function to graph csv data for text segment size and instructions counts.
```

Comment:
Use `"""` instead of `'''` for doc strings. The format should also be consistent with how function docstrings are traditionally done.

https://google.github.io/styleguide/pyguide.html#382-modules